ui/src/model-manager/install-models/InatallModelsModal.tsx

```diff
@@ -237,7 +246,7 @@ export default function InatallModelsModal({
                 return (
                   <ModelCard
                     model={model}
-                    key={model.id}
+                    key={model.id + apiKey}
```

Comment:
i don't think it's a good idea to put apikey in key, cuz we should keep this as secret as possible

Reply:
`ModelCard` memoizes `onClickInstallModel` function as prop, so in turn `apikey` also gets memoized, so it will never get updated, that's why I added it as key to force refresh it.
I'm open to suggestions on how to deal with this more elegently.